src/test/kotlin/uk/gov/justice/digital/assessments/services/AssessmentUpdateServiceOASysTest.kt

```diff
@@ -356,6 +358,7 @@ class AssessmentUpdateServiceOASysTest {
       )
     )
     every { assessmentUpdateRestClient.updateAssessment(any(), any(), any(), any()) } returns oasysError
+    every { predictorService.getPredictorResults(any(), any()) } returns emptyList()
```

Comment:
if possible i would avoid using any() for mocks as tests are more robust if we specify the right object inputs when mocking

Reply:
Tidied up this test 👍